docs/contributing/frontend.md

```diff
@@ -0,0 +1,133 @@
+# Frontend
+
+This is a guide to help the Coder community and also Coder members contribute to our UI. It is ongoing work but we hope it provides some useful information to get started. If you have any questions or need help, please send us a message on our [Discord server](https://discord.com/invite/coder). We'll be happy to help you.
+
+## Tech Stack
+
+All our dependencies are described in `site/package.json` but here are the most important ones:
+
+- [React](https://reactjs.org/) as framework
+- [Typescript](https://www.typescriptlang.org/) to keep our sanity
+- [Vite](https://vitejs.dev/) to build the project
+- [Material V4](https://v4.mui.com/) for UI components
+- [react-router](https://reactrouter.com/en/main) for routing
+- [TanStack Query v4](https://tanstack.com/query/v4/docs/react/overview) for fetching data
+- [XState](https://xstate.js.org/docs/) for handling complex flows
+- [axios](https://github.com/axios/axios) as fetching lib
+- [Playwright](https://playwright.dev/) for E2E testing
+- [Jest](https://jestjs.io/) for integration testing
+- [Storybook](https://storybook.js.org/) and [Chromatic](https://www.chromatic.com/) for visual testing
+
+## Structure
+
+All the code related to the UI is inside the `site`` folder and we defined a few conventions to help people to navigate through it.
+
+- **e2e** - E2E tests
+- **src** - Source code
```

Comment:
We could turn these into links [src](../../site/src) (not sure about the syntax though). I think that'll let markdown link check catch if any of these links every break due to restructuring, and it's a convenient way to hop in.

Reply:
Good one! Annotated to improve this in a next PR.